NativeApp/src/utilities/AxiosInstance.js

```diff
@@ -47,6 +48,17 @@ instance.interceptors.response.use((response) => {
       data: updatedMonthEvents,
     };
   }
+
+  if (response.config.url.includes(`${baseURL}/dashboard/getEmployeeTeamSnapshot`)) {
+    const events = [...response.data];
```

Comment:
Should this be `const events = [...response.data.events];` as above?

Reply:
no,  it's not

Reply:
Go team